lib/rspec/matchers.rb

```diff
@@ -927,6 +928,18 @@ def method_missing(method, *args, &block)
       end
     end
 
+    if RUBY_VERSION.to_f >= 1.9
+      def respond_to_missing?(method, include_private=false)
+        method = method.to_s
```

Comment:
The `to_s` here is unnecessary -- on 1.9+, symbols can be compared to regexes with `=~`.